lib/api_repo.js

```diff
@@ -56,6 +56,9 @@ var DEFAULT_PROTO_COMPILER = 'protoc';
 // Python builds using grpcio-tools.
 var NO_PROTOC_PLUGIN = ['nodejs', 'java', 'python'];
 
+// common protos for which we build gRPC service methods
+var COMMON_GRPC_PROTOS = ['operations.proto'];
```

Comment:
Adding bytestream.proto as well?  It's not in the common protos right now, but might be worth adding here.